tests/contrib/trace/conftest.py

```diff
@@ -18,6 +19,27 @@ def iopipe_with_trace_no_auto_measure():
     return IOpipe(token='test-suite', url='https://metrics-api.iopipe.com', debug=True, plugins=[plugin])
 
 
+@pytest.fixture
+def iopipe_with_trace_auto_http():
+    plugin = TracePlugin(auto_http=True)
+    return IOpipe(token='test-suite', url='https://metrics-api.iopipe.com', debug=True, plugins=[plugin])
```

Comment:
This and the next test are the same -- looks like you want to test an http endpoint here (perhaps good old `http://example.com`?)

Reply:
It's actually the handler fixture that makes the `http://` request. there may be a redundant fixture here, though.

Reply:
Update this to resolve the redundancy, please?

Reply:
This comment is done I believe, commenting so I remember myself